File: java/link-message-client/src/main/java/link/message/client/content/StringContent.java
package link.message.client.content;

import link.message.client.StringOrObjectSerializable;

/**
 * 直接把字符串作为content
 * @author zhongt
 *
 */
public class StringContent implements StringOrObjectSerializable {
	private String text;
	
	public StringContent(String text) {
		this.text = text;
	}
	
	@Override
	public String toString() {
		return text;
	}
}

File: net/Link.Message.Client/content/StringContent.cs
using System;
using System.CodeDom;
using Newtonsoft.Json;

namespace Link.Message.Client.content
{

	/// <summary>
	/// 直接把字符串作为content
	/// </summary>
	[JsonConverter(typeof(StringOrObjectConverter))]
	public class StringContent : IStringOrObjectSerializable
	{
		public string Text { set; get; }

	    public StringContent(String content)
	    {
	        Text = content;
	    }

	    public override string ToString()
	    {
	        return Text;
	    }
	}
}